config/development.js

```diff
@@ -3,6 +3,9 @@ exports.config = {
   common: {
     database: {
       name: process.env.DB_NAME_DEV
+    },
+    witter_api: {
+      url: process.env.WITTER_API_URL
```

Comment:
update .env.example with this new env var, so new people who download the project will know which URL is needed

Reply:
ok